analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Partial.razor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+namespace EmptyProject
+{
+    public partial class BlazorQueryParameterRoutableComponent_Partial : ComponentBase
+    {
+        [Parameter]
+        [SupplyParameterFromQuery]
+        public int Number { get; set; } // Compliant
+
+        [Parameter]
+        [SupplyParameterFromQuery]
```

Comment:
I think we should keep the information about S6803: 
`// S6803 Compliant: the .razor part of the component defines the route`